packages/suite/src/components/suite/NavigationBar/components/NavigationActions/components/EarlyAccessDropdown/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { Translation } from '@suite-components';
+import { Dropdown, DropdownRef, Icon, useTheme, variables } from '@trezor/components';
+import React, { useRef, useState } from 'react';
+import styled from 'styled-components';
+import ActionItem from '../ActionItem';
+import { useActions } from '@suite-hooks';
+import * as routerActions from '@suite-actions/routerActions';
+
+const Wrapper = styled.div<Pick<Props, 'marginLeft'>>`
+    ${props => props.marginLeft && `margin-left: 8px`};
+    position: relative;
+`;
+
+const Status = styled.button`
+    cursor: pointer;
+    border: 0;
+    background: none;
+    display: flex;
+    align-items: center;
+    text-align: left;
+    width: 100%;
+    & > * + * {
+        margin-left: 10px;
+    }
+`;
+
+const Details = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+const Label = styled.div`
+    font-size: ${variables.FONT_SIZE.NORMAL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    color: ${props => props.theme.TYPE_DARK_GREY};
+`;
+
+const GoToSettings = styled.div`
+    display: flex;
+    align-items: center;
+    background: none;
+    border: 0;
+    width: 16px;
+    margin: 0 10px 0 auto;
+`;
+
+interface Props {
+    marginLeft?: boolean;
+    isActive?: boolean;
+}
+
+const EarlyAccessDropdown = (props: Props) => {
+    const [open, setOpen] = useState(false);
+    const dropdownRef = useRef<DropdownRef>();
+    const theme = useTheme();
+    const { goto } = useActions({
+        goto: routerActions.goto,
+    });
+
+    return (
+        <Wrapper {...props}>
+            <Dropdown
```

Comment:
This should be improved. imho the design is weird 😀

Reply:
Design is weird, but not that important. Or do you mean the code structure? I couldn't agree more, I would like to refactor the navigation actions components...